lib/network.js

```diff
@@ -6,14 +6,6 @@ module.exports = function (archive, opts) {
   opts = opts || {}
 
   var swarm = hyperdiscovery(archive, opts)
-  var network = {
-    swarm: swarm,
-    options: swarm.options || opts
-  }
-
-  network.__defineGetter__('connected', function () {
-    return swarm.connected
-  })
-
-  return network
+  swarm.options = swarm.options || opts
```

Comment:
Darn. I can't remember why I put this in there.

Looks like hyperdiscovery doesn't set `swarm.options` and discovery-swarm sets `opts` to `swarm._options`. I probably just saw it was null in some situation so used this. But we should probably expose `swarm._options` directly here instead.